test/e2e/service_bootstrap.py

```diff
@@ -55,6 +55,11 @@ def create_execution_role() -> str:
     iam.attach_role_policy(
         RoleName=role_name, PolicyArn="arn:aws:iam::aws:policy/AmazonS3FullAccess"
     )
+    # This policy accommodates use of the feature group controller.
+    iam.attach_role_policy(
+        RoleName=role_name,
+        PolicyArn="arn:aws:iam::aws:policy/AmazonSageMakerFeatureStoreAccess"
```

Comment:
did you check what is present in this policy that is not in SagemakerFullAccess?

Reply:
I examined the two policies and included in the [ACK Feature Group](https://quip-amazon.com/MNHYAu4tEaRT/ACK-Feature-Group-Updated#QMc9CAmH6aG) quip doc what the SageMakerFeatureStoreAccess has that SageMakerFullAccess does not.

Reply:
looked at it and I think you shouldn't need it because [sagemaker execution role has S3 full access](https://github.com/aws-controllers-k8s/sagemaker-controller/blob/main/test/e2e/service_bootstrap.py#L55-L57)

please remove it and see